pygeo/parameterization/designVars.py

```diff
@@ -229,12 +229,6 @@ def __init__(self, name, lower, upper, scale, axis, coefListIn, mask, config, se
         super().__init__(name=name, value=np.zeros(nVal, "D"), nVal=nVal, lower=None, upper=None, scale=scale)
 
         self.config = config
-        if lower is not None:
```

Comment:
why are you removing these lines?

Reply:
That portion is duplicated in the class it inherits from, `geoDV`, so it shouldn't be necessary here.